src/cpu-template-helper/src/utils/mod.rs

```diff
@@ -0,0 +1,238 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::ffi::OsString;
+use std::fmt::{Binary, Display};
+use std::hash::Hash;
+use std::ops::{BitAnd, Shl};
+use std::path::{Path, PathBuf};
+use std::sync::{Arc, Mutex};
+
+use vmm::builder::{build_microvm_for_boot, StartMicrovmError};
+use vmm::resources::VmResources;
+use vmm::seccomp_filters::{get_filters, SeccompConfig};
+use vmm::vmm_config::instance_info::{InstanceInfo, VmState};
+use vmm::{EventManager, Vmm, HTTP_MAX_PAYLOAD_SIZE};
+
+#[cfg(target_arch = "aarch64")]
+pub mod aarch64;
+#[cfg(target_arch = "x86_64")]
+pub mod x86_64;
+
+const CPU_TEMPLATE_HELPER_VERSION: &str = env!("FIRECRACKER_VERSION");
```

Comment:
Do we really need this?It is only used to set `vmm_version` in `InstanceInfo`. Can't we just use some dummy string?

Reply:
We will use this information in the host fingerprint dump.

Reply:
I used the same way as the jailer.

Reply:
The `build.rs` set the env variable `FIRECRACKER_VERSION` to set the same number to all the binaries.
https://github.com/firecracker-microvm/firecracker/blob/a183614a6ce96ed61756f5e889e1396c51a61dc6/build.rs#L7

Reply:
Ok, let's let's leave this as it is for now.